app/controllers/Application.scala

```diff
@@ -73,31 +84,38 @@ class Application @Inject() (
   private final val MacAlgorithm = "HmacSHA1"
   private final val GitHubSignature = "X-Hub-Signature"
 
-  private val docRenderers = Map("conductr" -> conductrDocRenderer)
+  private val docRenderers = Map(
+    "conductr" -> Map(
+      "" -> conductrDocRenderer,
+      "1.0.x" -> conductrDocRenderer10,
+      "1.1.x" -> conductrDocRenderer11)
+  )
 
   private val secret = new SecretKeySpec(settings.play.crypto.secret.getBytes, MacAlgorithm)
 
+  private val webhookRef = (JsPath \ "ref").read[String]
```

Comment:
Would put `secret` and `webhookRef` close to the point where it is used. In this case above the `update` method. This is more readable when looking the first time on it.